src/Microsoft.DocAsCode.Build.Engine/CompilePhaseHandlerWithIncremental.cs

```diff
@@ -123,7 +123,11 @@ private void ReloadDependency(IEnumerable<HostService> hostServices)
                 var ldg = LastBuildVersionInfo?.Dependency;
                 if (ldg != null)
                 {
-                    // to-do: need reload dependency reference info.
+                    CurrentBuildVersionInfo.Dependency.ReportReference(from r in ldg.ReferenceReportedBys
```

Comment:
report reference  and report dependency are two different methods.

---
In reply to: [104120189](https://github.com/dotnet/docfx/pull/1383#discussion_r104120189) [](ancestors = 104120189)